src/layouts/WalletNotFoundLayout/WalletNotFoundLayout.scss

```diff
@@ -0,0 +1,47 @@
+@import '~@/scss/GlobalVariables';
```

Comment:
This scss file and the corresponding layout file has been moved inside the interface layout folder. You should rebase or pull down the current version or remove this for duplication.